RealtimeStreamToImpala/README.rst

```diff
@@ -0,0 +1,54 @@
+Realtime Stream To Impala Adapter Configuration
```

Comment:
@jwang47 @abaranau We will add a top level README that contains the getting CDAP SDK part, running shell